cmd/server/cadence/server_test.go

```diff
@@ -29,18 +29,16 @@ import (
 	"testing"
 	"time"
 
-	"github.com/uber/cadence/common/persistence/nosql/nosqlplugin/cassandra/gocql"
-
-	"github.com/uber/cadence/testflags"
-
 	"github.com/stretchr/testify/require"
 	"github.com/stretchr/testify/suite"
 
 	"github.com/uber/cadence/common"
 	"github.com/uber/cadence/common/config"
-	_ "github.com/uber/cadence/common/persistence/nosql/nosqlplugin/cassandra"              // needed to load cassandra plugin
+	_ "github.com/uber/cadence/common/persistence/nosql/nosqlplugin/cassandra" // needed to load cassandra plugin
```

Comment:
there is an option to group side-effect imports like this, which might be worth doing...

Reply:
yea, I think this is nicer.  especially with gci, which collapses slightly more.